reconcile/ocp_release_mirror.py

```diff
@@ -363,3 +367,6 @@ def run(dry_run: bool) -> None:
         except OcpReleaseMirrorError as details:
             LOG.error(str(details))
             sys.exit(ExitCodes.ERROR)
+        finally:
+            if quay_mirror is not None:
```

Comment:
Why this `if` statement?

Reply:
not needed, I'll remove it, thanks!

Reply:
this is still needed, but need to be done differently, `__init__` can raise error, so need to check if `quay_mirror` exist